yarn-project/cli/src/cmds/devnet/bootstrap_network.ts

```diff
@@ -134,8 +137,12 @@ async function deployToken(
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore - Importing noir-contracts.js even in devDeps results in a circular dependency error. Need to ignore because this line doesn't cause an error in a dev environment
   const { TokenContract, TokenBridgeContract } = await import('@aztec/noir-contracts.js');
-  const devCoin = await TokenContract.deploy(wallet, wallet.getAddress(), 'DevCoin', 'DEV', 18).send().deployed();
-  const bridge = await TokenBridgeContract.deploy(wallet, devCoin.address, l1Portal).send().deployed();
+  const devCoin = await TokenContract.deploy(wallet, wallet.getAddress(), 'DevCoin', 'DEV', 18)
+    .send({ universalDeploy: true })
```

Comment:
sneaky :laughing:

Reply:
ah wait this isn't necessary, was trying this before Phil's change on storing `initHash` & `salt`